catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/integration/test/CatalogIcebergIT.java

```diff
@@ -439,6 +439,61 @@ void testCreateAndLoadIcebergTable() {
                     sortOrders));
   }
 
+  @Test
+  void testTimestampTypeConversion() {
+
+    Column col1 =
+        Column.of("iceberg_column_1", Types.TimestampType.withTimeZone(), "col_1_comment");
+    Column col2 =
+        Column.of("iceberg_column_2", Types.TimestampType.withoutTimeZone(), "col_2_comment");
+
+    Column[] columns = new Column[] {col1, col2};
+
+    String timestampTableName = "timestamp_table";
+
+    NameIdentifier tableIdentifier =
+        NameIdentifier.of(metalakeName, catalogName, schemaName, timestampTableName);
+
+    Map<String, String> properties = createProperties();
+    TableCatalog tableCatalog = catalog.asTableCatalog();
+    Table createdTable =
+        tableCatalog.createTable(tableIdentifier, columns, table_comment, properties);
+    Assertions.assertEquals("iceberg_column_1", createdTable.columns()[0].name());
+    Assertions.assertEquals(
+        Types.TimestampType.withTimeZone(), createdTable.columns()[0].dataType());
+    Assertions.assertEquals("col_1_comment", createdTable.columns()[0].comment());
+
+    Assertions.assertEquals("iceberg_column_2", createdTable.columns()[1].name());
+    Assertions.assertEquals(
+        Types.TimestampType.withoutTimeZone(), createdTable.columns()[1].dataType());
+    Assertions.assertEquals("col_2_comment", createdTable.columns()[1].comment());
+
+    Table loadTable = tableCatalog.loadTable(tableIdentifier);
+    Assertions.assertEquals("iceberg_column_1", loadTable.columns()[0].name());
+    Assertions.assertEquals(Types.TimestampType.withTimeZone(), loadTable.columns()[0].dataType());
+    Assertions.assertEquals("col_1_comment", loadTable.columns()[0].comment());
+
+    Assertions.assertEquals("iceberg_column_2", loadTable.columns()[1].name());
+    Assertions.assertEquals(
+        Types.TimestampType.withoutTimeZone(), loadTable.columns()[1].dataType());
+    Assertions.assertEquals("col_2_comment", loadTable.columns()[1].comment());
+
+    org.apache.iceberg.Table table =
```

Comment:
why check hive catalog here?

Reply:
i want to check that the timestamp type conversion between gravitino and iceberg is correct

Reply:
ok